app/src/main/AndroidManifest.xml

```diff
@@ -1,5 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Revert all unwanted changes from this file. Change only that line which is required.